### ISSUE TITLE
Add Iinitial setup for uls-library-cornell-edu

### ISSUE THREAD

**Issue by manolobevia** [cul-it/uls#895]
- [x] Activate all plugins
- [x] Add CULU-CHILD theme
- [x] Create Main and Footer menus
- [x] Add basic configuration via customizer ( ex: hero background images)
- [x] Add Distributor plugin
- [x] Create distributor connections to MANN and OLINURIS libraries for staff development
- [x] Pull staff from mann and olinuris
- [x] Add ACF 2 WP API plugin
- [x] Add Application passwords plugin

**Comment by manolobevia** [comment]
For this initial setup, I had to create a local install. On dev I was getting a WSoD after enabling Elementor PRO plugin. I tried to remove the current plugin and add a new update, but I was getting this message: `Unable to create directory uploads/2020/10. Is its parent directory writable by the server?`. Locally I didn't have this issue. I was able to activate all plugins without any issue, after that I pushed all my DB changes to DEV.
